lib/exercises.js

```diff
@@ -1,11 +1,30 @@
 // This method will return an array of arrays.
 // Each subarray will have strings which are anagrams of each other
-// Time Complexity: ?
-// Space Complexity: ?
+// Time Complexity: O(n ^ 2) since the array being sorted has a length less than 10, the sort() method uses InsertionSort.
```

Comment:
But you also have a number of words, so you need to consider the array length as well.

If the words are small you could elect to consider sorting the words as a constant since it's pretty fast, but you **definitely** need to consider the number of strings.